ccd-definition/ComplexTypes/LitigationFriend.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "required",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is the claimant a child?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "fullName",
```

Comment:
why is this not split out with the name split that an individual has?

Reply:
A q for design and business

Reply:
I'll ask the team